src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/MQTT_JSON/MQTT_JSON.cpp

```diff
@@ -0,0 +1,123 @@
+#include <MQTT_JSON.hpp>
+
+void callback(char *topic, byte *payload, unsigned int length)
+{/*
```

Comment:
Don't commit code, that is in comment blocks, unless it is super-necessary. In version control you can always find the previous code in the commit history.